app/uk/gov/hmrc/hmrcemailrenderer/templates/onlinepaymentservice/AmountFormatter.scala

```diff
@@ -32,6 +32,7 @@ object AmountFormatter {
   }
 
   def formatAmount(amount: String): String = formatter.format(toDouble(amount))
+  def formatAmountPennies(amount: String): String = formatter.format(toDouble(amount)/100)
```

Comment:
Should be formatAmountPence or formatAmountInPence

Reply:
It's for formmating an amount that is in pennies to be shown in pounds

Reply:
I changed it please merge

Reply:
pennies still sounds wrong, please suggest something else

Reply:
I changed it check the latest version

Reply:
👍 good for me, you will need to change the other places before I merge

Reply:
Lol done now

Reply:
:-)